### PR TITLE
test_runner: add links to grafana for remote tests

### PR DESCRIPTION
**Pull request by bayandin**
## Describe your changes
Add Grafana links to the allure report to make it easier to debug perf test failures

- [Example in logs](https://github.com/neondatabase/neon/actions/runs/4627776430/jobs/8186091457#step:9:252)
- [Example in allure report](https://neon-github-public-dev.s3.amazonaws.com/reports/branch-bayandin-add-grafana-links-to-test-results/remote/4627776430/index.html#suites/5008d72a1ba3c0d618a030a938fc035c/928f1cee4ab72524/) 

## Issue ticket number and link
N/A

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.
- [ ] Do we need to implement analytics? if so did you add the relevant metrics to the dashboard?
- [ ] If this PR requires public announcement, mark it with /release-notes label and add several sentences in this section.

## Checklist before merging

- [ ] Do not forget to reformat commit message to not include the above checklist

### CODE REVIEW

File: test_runner/fixtures/neon_fixtures.py [hunk 1]
```diff
@@ -2436,10 +2438,15 @@ def remote_pg(
     connstr = os.getenv("BENCHMARK_CONNSTR")
     if connstr is None:
         raise ValueError("no connstr provided, use BENCHMARK_CONNSTR environment variable")
-
+    start_ms = int(datetime.timestamp(datetime.now(timezone.utc)) * 1000)
```
Comment (vadim2404) [inline test_runner/fixtures/neon_fixtures.py:2441]:
```python
int(datetime.utcnow().timestamp()) * 1000
```

it's a bit shorter

File: test_runner/fixtures/neon_fixtures.py [hunk 2]
```diff
@@ -2436,10 +2438,15 @@ def remote_pg(
...
     with RemotePostgres(pg_bin, connstr) as remote_pg:
         yield remote_pg
 
+    end_ms = int(datetime.timestamp(datetime.now(timezone.utc)) * 1000)
```
Comment (vadim2404) [inline test_runner/fixtures/neon_fixtures.py:2445]: the same

File: test_runner/fixtures/utils.py [hunk 3]
```diff
@@ -184,6 +186,47 @@ def allure_attach_from_dir(dir: Path):
...
 
+def allure_add_grafana_links(host: str, start_ms: int, end_ms: int):
+    """Add links to server logs in Grafana to Allure report"""
+    # We expect host to be in format like ep-divine-night-159320.us-east-2.aws.neon.build
+    endpoint_id, region_id, *_ = host.split(".")
+    # Add 10s margin to the start and end times
+    start_ms_str = str(start_ms - 10_000)
+    end_ms_str = str(end_ms + 10_000)
+
+    datasource = "xHHYY0dVz"
```
Comment (vadim2404) [inline test_runner/fixtures/utils.py:197]: it looks like a constant

File: test_runner/fixtures/neon_fixtures.py (hunk 1 above)
Comment (vadim2404) [reply test_runner/fixtures/neon_fixtures.py:2441]: and maybe it's better to multiple by `1_000` before coercing to `int`?

File: test_runner/fixtures/utils.py [hunk 4]
```diff
@@ -184,6 +186,47 @@ def allure_attach_from_dir(dir: Path):
             allure.attach.file(source, name, attachment_type, extension)
 
 
+def allure_add_grafana_links(host: str, start_ms: int, end_ms: int):
+    """Add links to server logs in Grafana to Allure report"""
+    # We expect host to be in format like ep-divine-night-159320.us-east-2.aws.neon.build
+    endpoint_id, region_id, *_ = host.split(".")
```
Comment (vadim2404) [inline test_runner/fixtures/utils.py:192]: NIT: 

```python
endpoint_id, region_id, _ = host.split(".", 2)
```

File: test_runner/fixtures/neon_fixtures.py (hunk 1 above)
Comment (bayandin) [reply test_runner/fixtures/neon_fixtures.py:2441]: > ```python
> int(datetime.utcnow().timestamp()) * 1000
> ```
> 
> it's a bit shorter

Yep, I was going to, but then I was scared by a huge red warning https://docs.python.org/3/library/datetime.html#datetime.datetime.utcnow 😄

Comment (vadim2404) [reply test_runner/fixtures/neon_fixtures.py:2441]: Hmm, I never saw errors with this. Nonetheless, there is a method `timestamp` in `datetime` object

**Review by vadim2404 — APPROVED** [review]

**Pull request merged**